### PR TITLE
revise explain format by list

### PR DESCRIPTION
**Pull request by shichun-0415**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v6.3 (TiDB 6.3 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/11185
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by Yisaer — APPROVED** [review]

File: sql-statements/sql-statement-explain.md [hunk 1]
```diff
@@ -185,9 +185,14 @@ EXPLAIN DELETE FROM t1 WHERE c1=3;
...
-If you do not specify the `FORMAT` in `EXPLAIN`, or specify `FORMAT = "row"`, `EXPLAIN` statement will output the results in a tabular format. See [Understand the Query Execution Plan](/explain-overview.md) for more information.
+| FORMAT | Description |
+| ------ | ------ |
+| Empty  | Same as row |
+| `row`  | The `EXPLAIN` statement outputs results in a tabular format. See [Understand the Query Execution Plan](/explain-overview.md) for more information. |
+| `brief`  | The operator IDs in the output of the `EXPLAIN` statement are simplified, compared with those when `FORMAT` is left unspecified. |
+| `dot`    | The `EXPLAIN` statement outputs DOT execution plans, which can be used to generate PNG files through a `dot` program (in the `graphviz` package). |
 
-If you specify `FORMAT = "brief"` in `EXPLAIN`, the operator IDs in the output are simplified compared with that without `FORMAT` specified.
+The following is an example when `FORMAT` is "brief"` in `EXPLAIN`:
```
Comment (Oreoxmt) [inline sql-statements/sql-statement-explain.md:195]:
```suggestion
The following is an example when `FORMAT` is `"brief"` in `EXPLAIN`:
```

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

**Pull request merged**